xarray/plot/plot.py

```diff
@@ -625,6 +641,10 @@ def imshow(x, y, z, ax, **kwargs):
     dimension can be interpreted as RGB or RGBA color channels and
     allows this dimension to be specified via the kwarg ``rgb=``.
 
+    Unlike matplotlib, Xarray can apply ``vmin`` and ``vmax`` to RGB or RGBA
+    data, by applying a single scaling factor and offset to all bands.
+    Passing  ``robust=True`` infers ``vmin`` and ``vmax`` in the usual way.
```

Comment:
... infers vmin and vmax from the 96-th percentile of the data distribution. (Or reference the “usual way” somehow).